### PR TITLE
Terese & Sofie - week 15

### PR DESCRIPTION
**Pull request by SofiePellegrini**
https://spacenotmaze.netlify.app/

### CODE REVIEW

File: code/public/index.html [hunk 1]
```diff
@@ -13,7 +16,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <link rel="icon" type="image/png" href="./alien.png" />
```
Comment (nadialefebvre) [inline code/public/index.html:19]: Nice favicon :)

File: code/src/components/Animation.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    position: absolute;
+    top: 10%;
+    left: 35%;
+  }
+  @media (min-width: 1400px) {
+    position: absolute;
+    top: 25%;
+    left: 40%;
+  }
+`;
```
Comment (nadialefebvre) [inline code/src/components/Animation.js:22]: Clear media queries

File: code/src/components/GameScreen.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  @media ${devices.desktop} {
+    width: 60vw;
+    font-size: 25px;
+    left: 20%;
+    top: 20%;
+    right: 20%;
+  }
+`;
+
+export const BtnNavigate = styled.button`
```
Comment (nadialefebvre) [inline code/src/components/GameScreen.js:40]: Impressive work to style your button!

File: code/src/components/StartScreen.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  }
+  @media ${devices.desktop} {
+    width: 400px;
+    height: 45px;
+  }
+`;
+
+const StartScreen = () => {
+  const dispatch = useDispatch();
+  const [username, setUsername] = useState('');
```
Comment (nadialefebvre) [inline code/src/components/StartScreen.js:55]: Maybe it's jus me, but I find it a bit confusing that the username with local state has the same name as the one in redux store. Maybe call the local one "usernameInput" or something like that?

File: code/src/components/StartScreen.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    console.log();
```
Comment (nadialefebvre) [inline code/src/components/StartScreen.js:59]: Oops, one console.log forgotten here :)

**Review by nadialefebvre — COMMENTED** [review]
Well done, Terese and Sofie, you can be proud! And I love the space theme :)

File: code/src/components/GameScreen.js [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+import { useSelector, useDispatch } from 'react-redux';
+import { continueGame } from '../reducers/labyrinth';
+import styled from 'styled-components';
+import EndOfGame from 'components/EndOfGame';
+
+export const devices = {
+  mobile: '(min-width: 375px)',
+  tablet: '(min-width: 768px)',
+  desktop: '(min-width: 1200px)',
+};
```
Comment (joannaringqvist) [inline code/src/components/GameScreen.js:11]: I agree, this was nice!

File: README.md [hunk 7]
```diff
@@ -1,13 +1,8 @@
...
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Week 15 project in the Technigo Bootcamp. 
+We were supposed to build a project which allows the user to navigate through a maze as a text-based adventure which is provided by a backend. It is built in Redux, using thunks. 
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://spacenotmaze.netlify.app/
```
Comment (joannaringqvist) [inline README.md:8]: Love the name ❤️ 👽

File: code/src/components/GameScreen.js [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+      #ff0000,
+      #ff7300,
+      #fffb00,
+      #48ff00,
+      #00ffd5,
+      #002bff,
+      #7a00ff,
+      #ff00c8,
+      #ff0000
+    );
```
Comment (joannaringqvist) [inline code/src/components/GameScreen.js:68]: Love your buttons!

**Review by joannaringqvist — COMMENTED** [review]
Great job with the Labyrinth! 🥳

**Pull request closed without merging**